playground/nuxt.config.ts

```diff
@@ -13,7 +17,31 @@ export default defineNuxtConfig({
     exposeConfig: true,
     cssPath: '~/assets/css/tailwind.css',
     editorSupport: true
-  },
+  } satisfies Partial<ModuleOptions>,
+  hooks: {
+    'tailwindcss:loadConfig': (config, configPath, idx) => {
+      logger.info('Running `tailwindcss:loadConfig` hook...', { configPath, idx })
+
+      if (idx === 0 && config) {
+        config.theme = config.theme ?? {}
+        config.theme.extend = config.theme.extend ?? {}
+        config.theme.extend.screens = { 'md2': '100px' }
+      } else if (idx === 1 && config) {
+        config.content = config.content ?? []
+        Array.isArray(config.content) ? config.content.push('my-content') : config.content.files.push('my-file-content')
+      }
+    },
+    'tailwindcss:config': (config) => {
+      logger.info('Running `tailwindcss:config` hook...')
+
+      config.theme = config.theme ?? {}
+      config.theme.extend = config.theme.extend ?? {}
+      config.theme.extend.colors = { extendedColor: '#000' }
```

Comment:
There are some conflicts here; `defu` will merge all `config.theme.extend.colors`, while our expose config has overridden this (unless it also imports config at runtime).

In this case, semantically, this should replace all `config.theme.extend.colors` than merge with it, might need the Proxy to put this in a function so defu can override.

Reply:
not an issue anymore